ONSdigital/eq-survey-runner
# Added JS for mutually exclusive checkboxes

### What is the context of this PR?
Adds the JS needed for the mutually exclusive checkbox to work. When JS is enabled, server-side validation won't be hit.
[Card](https://trello.com/c/aCpgM0A6/2177-eq-survey-runner-1627-completion-status-incorrectly-cleared-issue) for context.
[Pattern Lib implementation](https://sdc-global-design-patterns.netlify.com/components/detail/mutually-exclusive.html)
Test Schema - `test_checkbox_mutually_exclusive.json`
### How to review 
- Ensure mutually exclusive checkbox works as expected.
- Check both the visible and voice-over functionality works on different browsers, IE 9 and above.
